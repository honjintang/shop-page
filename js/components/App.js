import React from 'react'
import Feed from '~/components/shop/Feed'
import Card from 'react-bootstrap/Card'

const App = () => (
    <>
        <Card.Header>
            <h1>Viv's Vintage Shop</h1>
        </Card.Header >
        <Feed />
    </>
)

export default App