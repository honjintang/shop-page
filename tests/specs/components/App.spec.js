import React from 'react'
import { shallow } from 'enzyme'
import App from '~/components/App'

describe('components/App', () => {
    it('matches snapshot', () => {
        const component = shallow(<App />)
        expect(component).toMatchSnapshot()
    })
})