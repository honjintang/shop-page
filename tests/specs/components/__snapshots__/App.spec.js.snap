// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/App matches snapshot 1`] = `
<Fragment>
  <CardHeader>
    <h1>
      Viv's Vintage Shop
    </h1>
  </CardHeader>
  <Connect(Feed) />
</Fragment>
`;
